layouts/partials/footer.html

```diff
@@ -51,7 +51,7 @@
     </div>
     <div class="row pt-3 pt-md-0">
       <div class="col-12 pt-4">
-        <p class="footer-text">Initially created by Dynatrace</p>
+        <p class="footer-text">Initially created by <a href="https://www.dyntrace.com"  class="footer-text">Dynatrace</a></p>
```

Comment:
I think we should add ` target="_blank" rel="noopener"` to the link

Reply:
added it